package.json

```diff
@@ -27,34 +28,47 @@
     ]
   },
   "dependencies": {
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.0",
```

Comment:
We can move these two deps to `devDependencies`.

Reply:
Just moved. I also updated `.eslintrc` to not trigger error when importing `devDependencies` into test and spec files.